site/gdocs/components/AllCharts.scss

```diff
@@ -1,6 +1,43 @@
 .article-block__all-charts {
     h1.h1-semibold {
-        margin-bottom: 40px;
+        margin: 0;
+        flex: 1 1 0%;
+        .deep-link {
+            // Any value smaller than -16 causes the h1 to wrap in Firefox
+            margin-left: -16px;
```

Comment:
I played with this in Firefox and the negative margin seems unnecessary. Are you sure we need it?